internal/handlers/handler_reset_password_step2.go

```diff
@@ -52,5 +56,62 @@ func ResetPasswordPost(ctx *middlewares.AutheliaCtx) {
 		return
 	}
 
-	ctx.ReplyOK()
+	defer ctx.ReplyOK()
```

Comment:
I feel like this line may lead to issues, ctx.ReplyOK() sets the body content to a static value with the `{"status": "OK"}` as the body, so deferring this will override the `{"status": "KO"}` replies from ctx.Error(). Was this done to solve an issue?

Reply:
TL;DR i'm too lazzy
After the line 59, the `func` must response with ` ctx.ReplyOK()`, even if it cannot be notified. that's why i used `defer`
just fixed it